build/patch/pre-autogen/libevent/common/000.patch

```diff
@@ -108,19 +79,49 @@ index 284341c..5b8b0a6 100644
 + * have removed arc4random_addrandom() since 1.26. Since then, other libcs
 + * followed suit (e.g. Android's own libc). But libevent's arc4random.c
 + * copy still implement arc4random_addrandom().
++ *
++ * See also:
++ *
++ * - https://github.com/measurement-kit/libevent/commit/8b275d967d7ffd95d5cc12069aef35669126c6d9
++ * - https://bugzilla.mozilla.org/show_bug.cgi?id=931354
++ * - https://bug931354.bmoattachments.org/attachment.cgi?id=829728
```

Comment:
@AntonioLangiu I've added to the patch more documentation that explains what is going on here, after digging back into the history and understanding why the patch was issued in the first place.